packages/opencensus-core/package.json

```diff
@@ -38,6 +37,16 @@
     "LICENSE",
     "README.md"
   ],
+  "nyc": {
```

Comment:
Optional: Can we move these configuration options in `.nycrc`, later on we can add coverage threshold and other options. Also this will make `package.json` more cleaner. WDYT?

Reply:
The official docs didn't offer that as an option: https://istanbul.js.org/docs/tutorials/typescript/ . I don't know if it is supported.

Reply:
Ok. I referred this link https://github.com/istanbuljs/nyc#configuring-nyc